lib/graphql_schema.rb

```diff
@@ -24,6 +24,10 @@ def types
     @types ||= @hash.fetch('types').map{ |type_hash| TypeDefinition.new(type_hash) }.sort_by(&:name)
   end
 
+  def types_by_name
```

Comment:
I had misunderstood your question before, I was suggesting exposing a `type` method:

```ruby
def type(name)
  types_by_name.fetch(name)
end
```

Just like what exists in the tests